packages/tizen_log/lib/tizen_log.dart

```diff
@@ -0,0 +1,183 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:flutter/foundation.dart';
+
+typedef _DlogNative = Void Function(
+    Int32, Pointer<Utf8>, Pointer<Utf8>, Pointer<Utf8>);
+typedef _Dlog = void Function(int, Pointer<Utf8>, Pointer<Utf8>, Pointer<Utf8>);
+
+/// Provides the ability to use Tizen's logging service, dlog.
+///
+/// Tizen dlog system defines six log priority levels, from lowest to highest:
+/// VERBOSE, DEBUG, INFO, WARN, ERROR, and FATAL. These priorities are used to
+/// semantically categorize certain log types so that it's easier to search and
+/// filter relevant logs while debugging or monitoring the app. For more
+/// information see:
+/// https://docs.tizen.org/application/native/guides/error/system-logs/
+class Log {
+  Log._();
+
+  static final DynamicLibrary _library = DynamicLibrary.open('libdlog.so.0');
+  static final _Dlog _dlogPrint =
+      _library.lookup<NativeFunction<_DlogNative>>('dlog_print').asFunction();
+  static final RegExp _stackTraceRegExp =
+      RegExp(r'^#(\d+)\s+(.+)\((.+):(\d+):(\d+)\)$', multiLine: true);
```

Comment:
I just found that `DEBUG_MODE` doesn't work when the app is running in release mode because the stack trace format is different (the "column number" part is missing in release mode). I think you don't need to parse that part.

Debug mode:
```
#0      Log._getStackFrameAt (package:tizen_log/tizen_log.dart:134:22)
#1      Log._log (package:tizen_log/tizen_log.dart:115:36)
#2      Log.warn (package:tizen_log/tizen_log.dart:84:5)
#3      MyApp.build.<anonymous closure> (package:tizen_log_example/main.dart:48:38)
#4      _InkResponseState._handleTap (package:flutter/src/material/ink_well.dart:989:21)
```

Output:
```sh
I/tizen_log_example(  624): main.dart: <anonymous closure>(44) > info message
W/tizen_log_example(  624): main.dart: <anonymous closure>(48) > warn message
```

Release mode:
```
#0      Log._getStackFrameAt (package:tizen_log/tizen_log.dart:134)
#1      Log._log (package:tizen_log/tizen_log.dart:115)
#2      Log.warn (package:tizen_log/tizen_log.dart:84)
#3      MyApp.build.<anonymous closure> (package:tizen_log_example/main.dart:48)
#4      _InkResponseState._handleTap (package:flutter/src/material/ink_well.dart:989)
```

Output:
```sh
I/tizen_log_example(32556): -: -(-) > info message
W/tizen_log_example(32556): -: -(-) > warn message
```

Reply:
Thank you for catching that. I have modified the regex and removed `column` property from `_StackFrame` class. I checked and now it also works in release mode.

Reply:
Thank you for the quick fix!